lib/src/base-styles/_nx-page.scss

```diff
@@ -11,8 +11,10 @@
   display: grid;
   grid-template-columns: auto 1fr auto;
   grid-template-areas:
-    "title tags actions"
-    "description description description";
+    "title sub-title actions"
+    "description description description"
+    "tags tags tags";
+  column-gap: $nx-spacing-md;
   margin: 0 0 $nx-spacing-l 0;
```

Comment:
The individual tags have 8px of bottom margin which adds to this

Reply:
> The individual tags have 8px of bottom margin which adds to this

Removed. https://github.com/sonatype/sonatype-react-shared-components/pull/363/commits/efd0b5bd7ecab9f0a1acd4d3bbdd6f895fffed5b

Reply:
That solution creates problems when there are enough tags to wrap.
![image](https://user-images.githubusercontent.com/3630091/115755587-800b8980-a36b-11eb-92bb-601c84618f8d.png)